cpp/src/io/csv/csv_reader.cu

```diff
@@ -639,7 +639,10 @@ gdf_error read_csv(csv_read_arg *args)
 		CUDA_TRY(cudaMemcpy(d_str_cols, h_str_cols, sizeof(string_pair *)	* stringColCount, cudaMemcpyHostToDevice));
 	}
 
-	for (int col = 0; col < raw_csv->num_active_cols; col++) {
+	for (int acol = 0,col=-1; acol < raw_csv->num_actual_cols; acol++) {
+	if(raw_csv->h_parseCol[acol]==false)
```

Comment:
Please fix incorrect indentation of these lines.

Reply:
Fixed